easybuild/easyconfigs/l/libutempter/libutempter-1.1.6.2-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'libutempter'
+version = '1.1.6.2'
+
+homepage = 'http://git.altlinux.org/people/ldv/packages/?p=libutempter.git'
+description = """libutempter is library that provides an interface for terminal
+ emulators such as screen and xterm to record user sessions to utmp and wtmp files."""
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+
+# download via http://git.altlinux.org/people/ldv/packages/?p=libutempter.git;a=snapshot;h=1.1.6-alt2;sf=tgz
+# and rename libutempter-1.1.6-alt2-3c6e555.tar.gz to libutempter-1.1.6.2.tar.gz
+sources = [{
+    'filename': SOURCE_TAR_GZ,
+    'extract_cmd': "tar xfvz %s --strip 1",
+}]
+
+checksums = ['b7e55b666c575d8f2d294c82655b6af21f0c7339625ba6707e5edc2883a2b09a']
```

Comment:
@SimonPinches Hmm, I'm getting a different checksum? `de1183edd2c1ec80f93a57bc8847f9dd5d47774b78cef6c437583e3c3dae1226`

Reply:
@boegel @SimonPinches Just saw a different checksum:
`8b683c355d3376b95a27180f9d710c9b06261fd68b9587c0b13fe443f267d4c5`

Reply:
@boegel  @verdurin If I repeat the download and rename, then I get a different checksum again: `198092c3044c5d51bc142411b0a55b75d57207b80d99f1ee785fa4760dfb47cd` 
Since the original filename seems to already contain a hash, I'm surprised it's continually changing...
How should we handle this?

Reply:
Ugh, that's pretty annoying... Maybe they are fingerprinting downloads or something?

Any differences between two consecutive downloads that have different checksums with unpack & `diff -ru dir1 dir2`?

Reply:
@boegel @verdurin The extracted tarballs are the same, it's the timestamps of the tar files that are then gzipped that causes the different hashes.  If I first unzip all the various `tgz` files then the `sha256sum` of the tarballs all agree.  Anyway to first unzip and then test the checksum?

Reply:
No, the checksums are computed on the source files as specified in `sources`.

But, we can make unzipping part of the instructions (including a comment to explain why),  and list `SOURCE_TAR` in `sources`?